jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -88,13 +89,15 @@ def __init__(self, *args, **kwargs):
     def _file_parser(path):
```

Comment:
The docs say that the `label` column must exist, but we don't actually enforce that so you are able to load with just the `sky_centroid` column. This results in the `Object ID` column = N/A, so I think we should add a condition here so we make sure that column also exists.

Reply:
Thank you for your comment! Since `sky_coord` is sufficient to display the data on the viewer, I have updated the documentation to reflect that the label column is now optional. Additionally, I implemented logic to assign default values to the `Object ID` column when the `label` column is not present.